content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -31,6 +31,7 @@ The .NET Tracer supports automatic instrumentation on the following .NET Core ve
 
 | Version              | Microsoft End of Life | Support level        | Package version      |
 | -------------------- | --------------------- | -------------------- | -------------------- |
+| .NET 8               |                       | [GA](#support-ga)    | latest (>= 2.41.0)   |
```

Comment:
_Next_ release I would say, as that's when we're actually running on GA bits 🙂 

```suggestion
| .NET 8               |                       | [GA](#support-ga)    | latest (>= 2.42.0)   |
```

Reply:
We probably want to hold off on publishing this until that version releases if that's the case

Reply:
So we've discussed this internally. We support .NET 8 with 2.41 even though we were testing on the RC build only and we haven't added new features. That said nothing should prevent customers to use 2.41, and this will prevent people to ask the support teams.
We will update to 2.42 when we release to be even safer but we can merge as is.